src/assets/grails-template.html

```diff
@@ -1,7 +1,7 @@
 <html>
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta name="layout" content="main" />
+    <meta name="layout" content="react" />
```

Comment:
I'm confused about this change. Why not keep `content="main"` here and add the `<g:gtag />` line to `main.gsp`? Is there a reason we need both `main.gsp` and `react.gsp`?

Reply:
Nah, we don't use main anywhere else as far as I know. I don't remember having any reservations about that so my apologies for the confusion.

I would be more concerned with adding react.gsp since we already have a partial template _react.gsp so it might get confusing to distinguish between the two.

Reply:
Ok, I'll change it back to `main.gsp`